ts/boot/configureStoreAndPersistor.ts

```diff
@@ -24,13 +24,14 @@ const persistConfig: PersistConfig = {
   key: "root",
   storage: AsyncStorage,
   /**
-   * Sections of the store that must be persisted and rehydrated.
+   * Sections of the store that must be persisted and rehydrated with this storage.
    */
-  whitelist: ["session", "onboarding", "notifications", "profile", "entities"]
+  whitelist: ["onboarding", "notifications", "profile", "entities"]
```

Comment:
should we store also the `PIN` in the keychain?

Reply:
PIN is already stored in the Keychain but we are using the Keychain API directly inside `/ts/utils/keychain.ts`. This means we are writing/reading directly from the Keychain and not using the redux store. If needed we can open a story to move the PIN in the redux store and use the Keychan secure storage introduced in this PR.

Reply:
I think it would be sensible to do that, to have one single, consistent way of dealing with secret data

Reply:
Story added https://www.pivotaltracker.com/story/show/158465069